cmake/ZeekSpicyAnalyzerSupport.cmake

```diff
@@ -49,40 +52,44 @@ function(spicy_add_analyzer)
         COMMENT "Compiling ${SPICY_ANALYZER_NAME} analyzer"
         COMMAND mkdir -p ${SPICY_MODULE_OUTPUT_DIR_BUILD}
         COMMAND spicyz -o ${OUTPUT} ${SPICYZ_FLAGS} ${SPICY_ANALYZER_SOURCES}
-        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-        )
+        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 
-    add_custom_target(${SPICY_ANALYZER_NAME} ALL DEPENDS ${OUTPUT})
+    add_custom_target(
+        ${SPICY_ANALYZER_NAME} ALL
+        DEPENDS ${OUTPUT}
+        COMMENT "Preparing dependencies of ${SPICY_ANALYZER_NAME}")
 
-    if ( SPICY_MODULE_OUTPUT_DIR_INSTALL )
+    if (SPICY_MODULE_OUTPUT_DIR_INSTALL)
         install(FILES ${OUTPUT} DESTINATION "${SPICY_MODULE_OUTPUT_DIR_INSTALL}")
-    endif()
+    endif ()
 
-    if ( SPICY_SCRIPTS_OUTPUT_DIR_INSTALL AND DEFINED SPICY_ANALYZER_SCRIPTS )
-        if ( NOT DEFINED SPICY_ANALYZER_PACKAGE_NAME )
+    if (SPICY_SCRIPTS_OUTPUT_DIR_INSTALL AND DEFINED SPICY_ANALYZER_SCRIPTS)
+        if (NOT DEFINED SPICY_ANALYZER_PACKAGE_NAME)
             message(FATAL_ERROR "SCRIPTS argument requires PACKAGE_NAME")
-        endif()
+        endif ()
         install(
             FILES ${SPICY_ANALYZER_SCRIPTS}
-            DESTINATION "${SPICY_SCRIPTS_OUTPUT_DIR_INSTALL}/${SPICY_ANALYZER_PACKAGE_NAME}/${NAME_LOWER}")
-    endif()
+            DESTINATION
+                "${SPICY_SCRIPTS_OUTPUT_DIR_INSTALL}/${SPICY_ANALYZER_PACKAGE_NAME}/${NAME_LOWER}")
+    endif ()
 
     get_property(tmp GLOBAL PROPERTY __spicy_included_analyzers)
     list(APPEND tmp "${SPICY_ANALYZER_NAME}")
     set_property(GLOBAL PROPERTY __spicy_included_analyzers "${tmp}")
-endfunction()
+endfunction ()
 
 # Flag that analyzer is *not* being built. This is purely informational:
 # the cmake output will contain a corresponding note. Arguments are the
 # name of the analyzers and a descriptive string explaining why it's
 # being skipped.
-function(spicy_skip_analyzer name reason)
+function (spicy_skip_analyzer name reason)
```

Comment:
This formatting is a a bit odd, but I guess there's nothing to do about it with `separate_ctrl_name_with_space` on.

Reply:
Yes, the formatter doesn't really seem to distinguish these since in CMake on the AST level `if ... endif` probably isn't much different from `function ... endfunction`.